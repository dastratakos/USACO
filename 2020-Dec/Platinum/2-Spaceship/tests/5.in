60 5 60
110011010101010101100010110101101010100011011111011011011110
100001011010111100111111110011011011011000011111111100110101
100111111101110101111111110111001100011001010111111001110100
111011110101111011011110111011110110111111101011101111001011
001010001111110111011000011110101001011110011111111010101101
010001011101100111010111111111010110111111010111111010111100
110010011111110000001011111101010110101000111001100011111110
101111101100111111010101011100100001110101001111000111110111
110010111111100011101011010111111011001101111011000011110111
101111101100110111011101111100011001110010011111111111111010
111010111111101111110111111100111101010101101111011111001110
000011001110111111110101101111001101110001111110011101010111
011011011110101111111010000000111011010111110100111110100101
110101111110000100101011011111111101001011001101011111101001
101011110111011011101111101101111101101111011001101110001101
111100010001001010111110011111111011001100101111111101111100
111101111100010110100111101101111111001100110101110011001111
010110111011110110111111111110001010101100001111111010111100
000100111111110101101011101011110101100111010101111111111101
100111101110011111111011011101110011011110111101110011110011
001111010111111010011101001111001011101111010011101001101011
011011111010111011000111011111110100101011110110111111101011
111010111011011111111010000100100011011010000111101100101011
111111111111110100011101011100101100001111110011110101111011
011111110111011011111001110110101110011111110010100010010101
100011110001110111010000111101111110010111111000011111100011
011000011000110111111101111001111111010010100001100100010001
111101101010110111110101101111001111011011110111011110111000
111010110011110101010011110101011101110111111011111101111011
101011110011101111011001010111101100111010011101011110101100
100101101110101000001011111011111101001000110101110011110110
100100100101001011001110111100001111110101010101101110011000
110001110111101100100101000110010111110101001011001011111011
010101100000110101101111110011111100010111001111101010000011
100111001111111011100111111101100011001101001010100011000101
111101110011011011101111101011010111011111111010111111101101
001010111101010010001101101111111110011100000111110111100001
001100001111010010000010001001110111110001110111101110100111
111011111111101011000001010111110100000001001101010111101110
111111011010111101111011001001110011010111001111110110100111
100001111111100011110000110111101110100001111111011010011101
001110100111111111111110101101011101110111101111101110111111
111010010111010101100111111101011111100101011111110001011010
111100111101000111010111111110011101010011011000111110001111
000111100010111111010111111101111111110111111111111011111101
011110011101111110001110111101101111110011111110110000111101
010001101101101000011011011101111110100011011101110000011011
001110111100101011011100111101100101101111111110101110111011
111101101111101111011001111101111110111100111111100111110110
011100010101100100110111111110011000011011001011111111111110
010100100010110111010011001001011101101001111101001111000111
010110111111000101000111010111100111010101001101011001010111
100101100111011111111101111101011110111110110111111100111011
111110011111001011110101101111110110100100111110111110011110
011010111101100111111111100101110100111010101111111101011011
111111111111001111000101111011110111111111111111010111111100
101111101100110101101011000111101110100001110001110001100110
111011110101011011111100001010100001110011110001011100101101
110110111101110110100001110111110101111111111111101111111111
111110001111110110011111111111111111110011011110101111111000
3 49 5 28
3 49 4 48
3 49 3 50
3 49 1 38
3 49 5 18
3 49 4 41
3 49 4 8
3 49 2 24
3 49 3 40
3 49 1 21
3 49 3 58
3 49 3 43
3 49 1 39
3 49 2 18
3 49 3 49
3 49 4 57
3 49 2 50
3 49 4 30
3 49 2 9
3 49 3 1
3 49 4 22
3 49 4 35
3 49 3 44
3 49 2 21
3 49 3 8
3 49 4 17
3 49 4 19
3 49 4 28
3 49 3 35
3 49 5 23
3 49 2 17
3 49 1 30
3 49 4 30
3 49 1 29
3 49 2 3
3 49 4 30
3 49 5 1
3 49 5 8
3 49 5 19
3 49 1 5
3 49 1 40
3 49 4 6
3 49 5 22
3 49 4 3
3 49 5 6
3 49 2 27
3 49 1 25
3 49 2 6
3 49 4 34
3 49 4 8
3 49 3 43
3 49 1 41
3 49 4 40
3 49 3 21
3 49 2 59
3 49 2 52
3 49 3 21
3 49 3 30
3 49 2 7
3 49 3 49
