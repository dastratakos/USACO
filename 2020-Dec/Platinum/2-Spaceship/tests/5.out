599624995
45526585
511492739
445266523
471070025
870078898
790145937
512272867
994850319
848709918
300944165
655311559
280190020
362336951
160057880
60438412
789467000
464653312
211956891
261448319
499944183
621657886
191771609
979467150
784438665
431872843
291057454
46418402
592298349
437938193
758086725
722343132
464653312
818048371
957668851
464653312
581177793
510536142
506423062
898894767
193903154
627519122
870287926
559699656
452731950
474993121
45251916
391876442
681977686
790145937
655311559
906026732
98568691
997929062
704608332
961782773
997929062
991372727
799136276
160057880
